src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig

```diff
@@ -5,36 +5,34 @@
-            {% if collector.countMissings %}
-                {% set status_color = "red" %}
-            {% elseif collector.countFallbacks %}
-                {% set status_color = "yellow" %}
-            {% endif %}
+            {{ include('@WebProfiler/Icon/translation.svg.twig') }}
+            {% set status_color = collector.countMissings ? 'red' : collector.countFallbacks ? 'yellow' : '' %}
             {% set error_count = collector.countMissings + collector.countFallbacks %}
-            <span class="sf-toolbar-status{% if status_color is defined %} sf-toolbar-status-{{ status_color }}{% endif %}">{{ error_count ?: collector.countdefines }}</span>
+            <span class="sf-toolbar-value">{{ error_count ?: collector.countdefines }}</span>
         {% endset %}
+
         {% set text %}
-            {% if collector.countMissings %}
-                <div class="sf-toolbar-info-piece">
-                    <b>Missing messages</b>
-                    <span class="sf-toolbar-status sf-toolbar-status-red">{{ collector.countMissings }}</span>
-                </div>
-            {% endif %}
-            {% if collector.countFallbacks %}
-                <div class="sf-toolbar-info-piece">
-                    <b>Fallback messages</b>
-                    <span class="sf-toolbar-status sf-toolbar-status-yellow">{{ collector.countFallbacks }}</span>
-                </div>
-            {% endif %}
-            {% if collector.countdefines %}
-                <div class="sf-toolbar-info-piece">
-                    <b>Defined messages</b>
-                    <span class="sf-toolbar-status sf-toolbar-status-green">{{ collector.countdefines }}</span>
-                </div>
-            {% endif %}
+            <div class="sf-toolbar-info-piece">
+                <b>Missing messages</b>
+                <span class="sf-toolbar-status sf-toolbar-status-{{ collector.countMissings ? 'red' }}">
+                    {{ collector.countMissings|default(0) }}
+                </span>
+            </div>
+
+            <div class="sf-toolbar-info-piece">
+                <b>Fallback messages</b>
+                <span class="sf-toolbar-status sf-toolbar-status-{{ collector.countFallbacks ? 'yellow' }}">
+                    {{ collector.countFallbacks|default(0) }}
+                </span>
+            </div>
+
+            <div class="sf-toolbar-info-piece">
+                <b>Defined messages</b>
```

Comment:
The title can be improved, but it's the same title that Symfony has used since the beginning and the profiler uses too. I guess it's better not to change it in this PR.